charts/incubator/kopia/questions.yaml

```diff
@@ -131,7 +132,7 @@ questions:
                   default: 568
               - variable: runAsGroup
                 label: "runAsGroup"
-                description: "The groupID of the user running the application"
+                description: "The groupID this App of the user running the application"
```

Comment:
Can you please the chart you copy this from?
The “this app” makes zero sense here and I see it on all you pra

Reply:
idk where its coming from :/
lots of apps have it -> https://github.com/search?q=repo%3Atruecharts%2Fcharts+The+groupID+this+App+&type=code